### ISSUE TITLE
Only works with pull requests

### ISSUE THREAD

**Issue by skrobinson** [JacobDomagala/StaticAnalysis#25]
It would be nice if StaticAnalysis would work with other actions as well as PR. I'm trying out SA and want to repeatedly trigger it while I make changes to the configuraton. Either `push` or manual triggers would help during setup.

**Comment by DevinSodhi** [comment]
I second this. For some reason, trying to run on push simply fails on the python script. It would be nice to associate a badge with a successful merge/push that shows a pass condition.

**Comment by JacobDomagala** [comment]
Yes, currently it only works on PR and this is intended. I suppose this action could be modified to also run on push (with the addition of input variable, or perhaps it could be detected on runtime, using GitHub's environment variables), and then it would just output the result to console/stdout. 

If I find some time in near future, I will try to implement that.
